src/reporter/tests/test_NTNENA.py

```diff
@@ -1189,3 +1190,213 @@ def test_aggregation_is_per_instance(service, reporter_dataset):
+    ]
+
+    expected = {
+        'attrs': expected_attrs
+    }
+
+    assert obtained == expected
+
+    # 'aggrMethod': 'max'
+
+    query_params = {
+        'attrs': attrs,
+        'aggrMethod': 'max',
+    }
+    r = query(params=query_params, service=service)
+    assert r.status_code == 200, r.text
+
+    obtained = r.json()
+    assert isinstance(obtained, dict)
+    expected_attrs = [
+        {
+            'attrName': 'boolean',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': [True]
+                }],
+                'entityType': 'Test'
+            }]
+        },
+        {
+            'attrName': 'intensity',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': ['Low']
+                }],
+                'entityType': 'Test'
+            }]
+        },
+        {
+            'attrName': 'temperature',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': [10.0]
+                }],
+                'entityType': 'Test'
+            }]
+        },
+    ]
+
+    expected = {
+        'attrs': expected_attrs
+    }
+    obtained = r.json()
+    assert obtained == expected
+
+    # 'aggrMethod': 'count'
+
+    query_params = {
+        'attrs': attrs,
+        'aggrMethod': 'count',
+    }
+    r = query(params=query_params, service=service)
+    assert r.status_code == 200, r.text
+
+    obtained = r.json()
+    assert isinstance(obtained, dict)
+    expected_attrs = [
+        {
+            'attrName': 'boolean',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': [1]
+                }],
+                'entityType': 'Test'
+            }]
+        },
+        {
+            'attrName': 'intensity',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': [1]
+                }],
+                'entityType': 'Test'
+            }]
+        },
+        {
+            'attrName': 'temperature',
+            'types':
+            [{
+                'entities':
+                [{
+                    'entityId': 'Test1',
+                    'index': ['',''],
+                    'values': [1]
+                }],
+                'entityType': 'Test'
+            }]
+        },
+    ]
+
+    expected = {
+        'attrs': expected_attrs
+    }
+
+    obtained = r.json()
+    assert obtained == expected
```

Comment:
count indeed should work

Reply:
yes it works, I shared in my comment https://github.com/orchestracities/ngsi-timeseries-api/issues/296#issuecomment-880033702